tasks/sep/fp-recovery/005-set-game-implementation/justfile

```
@@ -94,3 +96,107 @@ set-implementation gameType='' implAddr='':
   INFO="* Set implementation for game type {{gameType}} to {{implAddr}} in \`DisputeGameFactory\` ${DGF_ADDR}: \`setImplementation({{gameType}}, {{implAddr}})\`"
   sed -e "s|<!--NEXT TASK DESCRIPTION-->|${INFO}\n<!--NEXT TASK DESCRIPTION-->|" out/README.md > out/README.md.tmp
   mv out/README.md.tmp out/README.md
+
+copy-anchor-state fromGameType='' toGameTypes='':
+  #!/usr/bin/env bash
+  set -euo pipefail
```

similar to above task
```suggestion
  set -euo pipefail
  unset ETH_RPC_URL
```